doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+            for the first five questions.
+        </div>
+        <div class="container">
+            <p><a class="btn btn-lg btn-success" href="fundamentals_quiz" role="button">Fundamentals Quiz</a></p>
+        </div>
+        <br/>
+        <h1><a id="S1">Using version control</a></h1>
+        <p>
+        <ul>
+            <li>
+                <h2>Introduction</h2>
+            </li>
+            <div class="page-header">
+                <h3>Why use a version control system?</h3>
+            </div>
+            <div class="well">
+                Any good version control system provides helpful tools to allow users to;
+                <ul>
+                    <li>share work and encourage collaboration,</li>
+                    <li>track the progression and incremental changes to versions/revisions of 
+                        work,
+                    </li>
+                    <li>allow multiple flavours of the some work to investigate the impact of a 
+                        change,
+                    </li>
+                    <li>traceability of work, for example what version of the work was used and 
+                        when,
+                    </li>
+                    <li>to revert a change,</li>
+                    <li>branch from a version, 
+                    <li>merge in changes</li>
+                    <li>and so much more.</li>
+                </ul>
+                <br/>
+            </div>
+            <div class="page-header">
+                <h3>Subversion</h3>
+            </div>
+            <div class="well">
+                Subversion has some very good documentation, Version Control with Subversion, 
+                referred to in this training material as the 
+                <a href="http://svnbook.red-bean.com/en/1.8/">Subversion book</a>. 
+                It is recommended that you read this as it provides a good introduction to 
+                the basic concepts of version control.
+                Please read <a href="http://svnbook.red-bean.com/en/1.8/svn.basic.html">Fundamental Concepts</a> 
+                if you are not already familiar with Subversion or CVS.
+                Specific topics to focus on are Copy-Modify-Merge for shared file systems and 
+                Global Revision Numbers.
+                <br/>
+                Note that Fcm working practices ensure that your working copy (a local copy of 
+                the repository's files and directories where you can prepare changes) always 
+                corresponds to exactly one revision.
+                <br/>
+                CVS users should have a basic understanding of version control but there are 
+                some differences and the section <a href="http://svnbook.red-bean.com/en/1.8/svn.forcvs.html">Subversion for CVS Users</a> was
+                written for users moving from CVS to Subversion (SVN).
+                <br/>
+                <br/>
+                <h4>Summary of svn:</h4>
```

Comment:
You might want to call this "Key Features of ..." rather than "Summary of" as you're not summarising anything you've said, rather highlighting it's capabilities.